shared/modules/network.utils.ts

```diff
@@ -91,3 +92,17 @@ export function convertNetworkId(value: unknown): string {
   }
   throw new Error(`Cannot parse as a valid network ID: '${value}'`);
 }
+
+/**
+ * Converting a hex chainId to decimal
+ *
+ * @param chainId - The chainId to be converted
+ * @returns The chainId as decimal
```

Comment:
This doc may need to be updated to reflect the 'returns chainId passed as parameter if it is a non-hex value' test case. Otherwise, looks great!

Reply:
done